components/logins/src/error.rs

```diff
@@ -141,9 +145,8 @@ impl GetErrorHandling for Error {
                 })
                 .report_error("logins-migration")
             }
-            Self::CryptoError(_) => {
-                ErrorHandling::convert(LoginsApiError::IncorrectKey).log_warning()
-            }
+            Self::CryptoError { .. } => ErrorHandling::convert(LoginsApiError::IncorrectKey)
+                .report_error("logins-crypto-error"),
```

Comment:
`IncorrectKey` may not be the the right error for the API.  It's possible that we're getting invalid ciphertext.  Once we get error reports back, we could consider changing it.

Reply:
I might be misunderstanding what happens here, is the `action` we add to the `CryptoError` reported?